.github/workflows/composite-build-sw-container.yml

```
@@ -0,0 +1,38 @@
+name: Build container images
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+    types:
+      - dispatch-build
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'opensciencegrid/')
+    strategy:
+      fail-fast: False
+      matrix:
+        repo: ['development', 'testing', 'release']
+        registry:
+          - url: hub.opensciencegrid.org
+            username: OSG_HARBOR_ROBOT_USER
+            password: OSG_HARBOR_ROBOT_PASSWORD
+          - url: docker.io
+            username: DOCKER_USERNAME
+            password: DOCKER_PASSWORD
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Push container
+      uses: opensciencegrid/push-container-action@v1
```

```suggestion
      uses: opensciencegrid/push-container-action@v0.2.0
```

As discussed, we'll tag your latest change as `0.2.0`